localstack/utils/aws/aws_stack.py

```diff
@@ -179,14 +179,10 @@ def is_internal_call_context(headers) -> bool:
 
 def get_local_service_url(service_name_or_port: Union[str, int]) -> str:
     """Return the local service URL for the given service name or port."""
+    # TODO(srw): we don't need to differentiate on service name any more, so remove the argument
```

Comment:
Is this something that will be done in a follow up or should this be part of this PR?

Reply:
Follow up